src/tools/kg_tools.py

```
@@ -21,71 +26,199 @@ async def you_trippin(ctx: Context, script_path: str) -> str:
     try:
+        await ctx.report_progress(progress=40, total=100, message="Executing Neo4j query...")
         async with repo_extractor.driver.session() as session:
             result = await session.run(command)
+            await ctx.report_progress(progress=80, total=100, message="Processing query results...")
             records = [dict(record) for record in await result.list()]
-            return json.dumps({"success": True, "data": records}, indent=2)
+            await ctx.report_progress(progress=100, total=100, message="Query completed successfully")
+            return create_success_response(
+                tool_name="graph_query",
+                data={"query": command, "results": records, "result_count": len(records)},
+                message=f"Query returned {len(records)} records"
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)